tests/acceptance/index-test.js

```diff
@@ -10,6 +12,23 @@ module('Acceptance | index', function(hooks) {
 
   hooks.beforeEach(function() {
     loadDefaultScenario(this.server);
+
+    this.clock = FakeTimers.install({
+      now: new Date('2020-11-05T20:59:54Z'),
```

Comment:
I chose a time that occurs a few seconds after `updated_at` for an issue titled `Add header and footer to application.hbs`. (I checked that there are no other issues in `mirage/data` with an even later `update_at`.)

https://github.com/ember-learn/ember-help-wanted/blob/6413e99895406077f5ba00e1e1a9b775bc8b8e3e/mirage/data/github-issues/ember-learn.js#L1516